wasm/validator/derive/src/conversion/asset_definition.rs

```diff
@@ -11,7 +11,7 @@ pub fn impl_derive_ref_into_asset_definition_owner(input: TokenStream) -> TokenS
     impl_from(
         &input.ident,
         &input.generics,
-        &syn::parse_quote!(::iroha_wasm::validator::pass_conditions::asset_definition::Owner),
+        &syn::parse_quote!(::iroha_validator::pass_conditions::asset_definition::Owner),
         &syn::parse_quote!(asset_definition_id),
     )
     .into()
```

Comment:
IMO `mod.rs`, `account.rs`, `asset.rs` and `asset_definition.rs` are too small to be in separate modules. I would just place all this code in `conversion.rs` and delete those